src/IProfile.jl

```diff
@@ -96,6 +96,7 @@ function insert_profile_block(fblock::Expr, tlast, tnow, timers, blast, bnow, by
     end
     descend = PROFILE_STATE & PROFILE_DESCEND > 0
     fblocknewargs = []
+    local lasttag
```

Comment:
Yeah, I think IProfile was written in the julia 0.1 days or so. I seem to remember some julia commit---long ago---that changed this.